src/main/java/edu/kis/powp/jobs2d/command/manager/DeviceUsageManager.java

```diff
@@ -0,0 +1,49 @@
+package edu.kis.powp.jobs2d.command.manager;
+
+
+import edu.kis.powp.observer.Publisher;
+
+/**
+ * Manager class for the device.
+ * Can be used by multiple drivers
+ */
+public class DeviceUsageManager {
+	private final Publisher publisher = new Publisher();
+
+	private double distanceTravelled = 0.0;
+	private double distanceDeviceUsed = 0.0;
+
+	public void notifyHeadMovePerformed(double distance) {
```

Comment:
'notify' is a collateral result, the operation is more about updating usage.

Reply:
renamed